soql-stdlib/src/main/scala/com/socrata/soql/functions/Docs.scala

```diff
@@ -0,0 +1,88 @@
+package com.socrata.soql.functions
+
+import com.socrata.soql.types._
+
+object Docs {
+    def main(args: Array[String]) {
+        val out_path = args(0)
+        SoQLFunctions.allFunctions.filter(x => x.name.toString != "op$[]").foreach { function => 
+            val doc = makeFuncDoc(function)
+            println(function.name.toString)
+            val cleanedName = cleanName(function.name.toString)
+            val file_path = out_path + s"$cleanedName.md"
+            println(file_path)
+            val f = new java.io.File(file_path)
+            if(!f.exists) f.createNewFile
+            val fw = new java.io.FileWriter(f)
```

Comment:
`FileWriter`'s api is :sob: , generally you want to specify the charset when writing data and `FileWriter` doesn't let you do that.  That's why everywhere I write files you see
```scala
for {
  fos <- managed(new FileOutputStream(file))
  osw <- managed(new OutputStreamWriter(fos, StandardCharsets.UTF_8))
} {
  ... actually write into osw ...
}
```